packages/react-next/src/components/Link/useLink.ts

```diff
@@ -0,0 +1,33 @@
+import { ComposePreparedOptions } from '@fluentui/react-compose';
+import { mergeProps } from '../../utils/mergeProps';
+import { LinkProps, LinkSlots, LinkSlotProps } from './Link.types';
+
+/**
+ * The useLink hook processes the Link component props and returns
+ * state, slots and slotProps for consumption by the component.
+ */
+export const useLink = (props: LinkProps, options: ComposePreparedOptions) => {
+  const { 'aria-describedby': ariaDescribedBy, as, disabled, href, keytipProps, onClick } = props;
+
+  const _onClick = (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => {
```

Comment:
```suggestion
  const onClick = (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => {
```

Reply:
I cannot name it `onClick` because I'm getting `onClick` from the props unless I rename the props `onClick` to something else. Naming suggestions appreciated.

Reply:
got it. i don't have a great suggestion. an alternative is to use `props.onClick` everywhere. but i think what you currently have is better?